modules/check_in/spec/services/map/service_spec.rb

```diff
@@ -41,7 +41,7 @@
       end
 
       it 'returns token from redis' do
-        expect(subject.build.token).to eq(access_token)
+        expect(subject.build(opts).token).to eq(access_token)
```

Comment:
@kanchanasuriya Hi, can you confirm this addition is okay and doesn't change the intent of the test?

If `opts` is intended to be left out, we need to come up with a code change to properly handle a change in Rials 7.1. According to the docs they're now raising an error if a key isn't passed to the `ActiveStorage::Cache` ([Changelog](https://github.com/rails/rails/blob/7-1-stable/activesupport/CHANGELOG.md?plain=1#L462): "Consistently raise an ArgumentError if the ActiveSupport::Cache key is blank.").

Depending on what the expected behavior is, we may need to rescue this error in the Map::Service. Let me know your thoughts and we can chat about it. Thanks!

Reply:
Changes look good, @ryan-mcneil.  Method response is anyway mocked and adding this parameter doesn't change any behaviour in the test.